code/game/gamemodes/cult/runes/conversion.dm

```diff
@@ -0,0 +1,87 @@
+					target.apply_effect(rand(1, 10), STUTTER)
+					target.take_overall_damage(0, 10)
+				if(75 to 100)
+					to_chat(target, span("cult", "Your mind turns to ash as the burning flames engulf your very soul and images of an unspeakable horror begin to bombard the last remnants of mental resistance."))
+					//broken mind - 5000 may seem like a lot I wanted the effect to really stand out for maxiumum losing-your-mind-spooky
+					//hallucination is reduced when the step off as well, provided they haven't hit the last stage...
+					target.hallucination += 5000
+					target.apply_effect(10, STUTTER)
+					target.adjustBrainLoss(1, 55)
+				if(100 to INFINITY)
+					to_chat(target, span("cult", "Your entire broken soul and being is engulfed in corruption and flames as your mind shatters away into nothing."))
+					target.hallucination += 5000
+					target.apply_effect(15, STUTTER)
+					target.adjustBrainLoss(rand(1, 5), 55)
+
+		initial_message = TRUE
+		if(!target.can_feel_pain())
+			target.visible_message(span("warning", "The markings below [target] glow a bloody red."))
+		else
+			target.visible_message(span("warning", "[target] writhes in pain as the markings below \him glow a bloody red."), span("danger", "AAAAAAHHHH!"))
+
+		if(!waiting_for_input[target]) //so we don't spam them with dialogs if they hesitate
+			waiting_for_input[target] = TRUE
+
+			if(!cult.can_become_antag(target.mind) || jobban_isbanned(target, "cultist"))//putting jobban check here because is_convertable uses mind as argument
+				//waiting_for_input ensures this is only shown once, so they basically auto-resist from here on out. They still need to find a way to get off the freaking rune if they don't want to burn to death, though.
+				to_chat(target, span("cult", "Your blood pulses. Your head throbs. The world goes red. All at once you are aware of a horrible, horrible truth. The veil of reality has been ripped away and in the festering wound left behind, something sinister takes root."))
+				to_chat(target, span("cult", "And you were able to force it out of your mind. Though the memory of that dark, horrible vision will surely haunt you for decades to come."))
+				var/has_implant // we don't want people with loy implants to just get gibbed
+				for(var/obj/item/implant/loyalty/L in target)
+					if(L?.imp_in == target)
+						has_implant = TRUE
+				if(!has_implant)
+					spawn(10)
+						to_chat(target, span("cult", "..or will it?"))
+						target.gib() // people who can't be cultists get gibbed to preserve cult anonymity
+			else spawn()
+				var/choice = alert(target,"Do you want to join the cult?", "Submit to Nar'Sie", "Resist", "Submit")
+				waiting_for_input[target] = FALSE
+				if(choice == "Submit") //choosing 'Resist' does nothing of course.
+					cult.add_antagonist(target.mind)
+					converting -= target
+					target.hallucination = FALSE //sudden clarity
+					target.rejuvenate() // nar'sie heals you
```

Comment:
Rather than rejuvenating them outright, consider just removing the brain damage.